spec/coffee/bootstrap.coffee

```diff
@@ -1,6 +1,48 @@
 #global jasmine protects
 beforeEach ->
   @googleTemp = window.google
+
+  angular.module('uiGmapgoogle-maps')
+  .config ($provide) ->
+    $provide.decorator '$timeout', ($delegate, $browser) ->
+      $delegate.hasPendingTasks = ->
+        $browser.deferredFns.length > 0
+
+      $delegate
+
+  #create
+  angular.module('uiGmapgoogle-maps.specs', ['uiGmapgoogle-maps'])
+  #require for testing
+  module "uiGmapgoogle-maps.specs"
+
+  @injectAll = =>
+    @injects.forEach (toInject) ->
```

Comment:
allow injections to be lazily loaded and initiated later. This way later modules can all be created prior to any injector's creation.